src/mt/router.hpp

```diff
@@ -0,0 +1,26 @@
+//
+// Created by Z1ko on 28/07/2022.
+//
+
+#ifndef FURRENT_ROUTER_HPP
```

Comment:
Although the functionality is the same, I think `#pragma once` would look cleaner and more consistent with the rest of the source.